source/docs/tuner/tuner-swerve/validating-drivetrain.rst

```diff
@@ -0,0 +1,73 @@
+Validating the Drivetrain
+=========================
+
+Tuner performs several validation procedures to confirm inverts, offsets and mechanical functionality of the drivetrain.
+
+This test consists of two steps:
+
+* Verify Steer
+* Verify Drive
+
+.. image:: images/swerve-verification-page.png
+   :alt: Picture of the verification page in Tuner
+
+Verify Steer
+------------
+
+.. image:: images/swerve-verify-steer.png
+   :alt: Picture of the steer verification page in Tuner
+
+The **Verify Steer** test is used to confirm that the module azimuth (or steer) can rotate freely and what direction they rotate in.
+
+This test simply rotates all of the modules and confirms with the user if they rotated clockwise or counter-clockwise. The modules should rotate **counter-clockwise**.
```

Comment:
Clockwise or counterclockwise as viewed from what direction?
If I'm looking at it from the bottom, that's different than looking at it from the top.